deployment_config/prod_vars.yml

```
@@ -1,6 +1,6 @@
 env: prod
-web_instances: 2
-web_memory: 512M
+web_instances: 3
+web_memory: 1GB
```

Let's do 2GB to give us wiggle room since we have ~888MB data. We have the memory capacity to do that for 2 or 3 instances for the short/medium term.

---

I read the docs as stating that the absolute limit to this value is 1GB. Your reading is probably more accurate.
```suggestion
web_memory: 2GB
```